.github/workflows/build-test.yml

```
@@ -231,16 +226,64 @@ jobs:
       - name: Run playwright tests
         env:
           BROWSER: ${{ matrix.browser }}
```

actionlint (1.7.10)

[error] property "browser" is not defined in object type {}

(expression)